Gemfile.lock

```diff
@@ -0,0 +1,147 @@
+GEM
```

Comment:
Are you sure we should add a `Gemfile.lock` while we depend on external service updates that are always in sync with the dynamic requires specified in the `Gemfile`?